### ISSUE TITLE
Enabled and using on Android

### ISSUE THREAD

**Issue by Randy-Astle** [pjeby/pane-relief#40]
I finally enabled Pane Relief on my Samsung tablet (Android). 

Superficially, it is working, but I haven't done any systematic testing.

If you can provide me with a list of features that you would like verified, I'm happy to test them for you.

Randy

**Comment by pjeby** [comment]
Well, all the features are in the command list, basically, apart from the behavior of the nav arrows (if you have tab headers turned on).  So if every command works, and the history works (shows numbers, has previewable/draggable menu items, and is saved across restarts) then I think that covers everything.

There are a *lot* of commands, though, for everything from maximizing to sliding to swapping tabs around to locking and unlocking the sidebars (that I'm not sure are even present on mobile).

**Comment by Randy-Astle** [comment]
Thanks -- I thought there might be a subset of the features that you thought were possibly more likely to not work on mobile.

Regardless, I'll create a list and work my way through it.

**Comment by pjeby** [comment]
Thank you!

**Comment by Randy-Astle** [comment]
I've done some initial experimenting with Pane Relief on my Samsung phone and on my Samsung tablet.

Somewhat to my surprise there are some differences in how it behaves on the phone vs the tablet.

A significant difference is in Settings | Appearance | Advanced:
- The desktop has 7 options
- The Android tablet has 3 options
   1. Show inline title
   2. Ribbon menu
   3. Show tab title bar
- The Android phone has 2 options
   1. Show inline title
   4. Ribbon menu

Does Obsidian allow plugins to differentiate between tablets and phones?

Some features will be handy on the tablet and not useful on a phone. (e.g. maximize active tab -- useful on the tablet, but on my phone the tab is already maximized, in effect.)

**Comment by pjeby** [comment]
I don't think plugins get that info directly, but then I'm not doing any mobile development currently.  Let me know if any of the features that are mobile-relevant don't work or behave differently than you would expect and I will see what I can do though.

**Comment by Randy-Astle** [comment]
Looking through the README.md, for Pane Relief, I noted that it still references your Hotkey Helper Plugin.

I used to use it, but my understanding is that it's functionality was Sherlocked into Obsidian. (So, I uninstalled it.) Does Hotkey Helper still add additional, useful functionality?

**Comment by pjeby** [comment]
Yes.  The sherlocking thing is *enormously* exaggerated and I would appreciate people not spreading that misinformation.

The only thing I'm aware that they added to Obsidian core was the hotkey/config *buttons* -- which is far from HH's only feature.  The changelog at the top of HH's readme lists plenty of other features that Obsidian does not have in core.

For example, Obsidian doesn't show you the counts or conflicts per plugin.  Obsidian's built-in hotkey search does not properly filter the hotkeys by plugin -- if one plugin's name contains the name of another plugin, you can end up viewing hotkeys for both.  Hotkey Helper also offers both commands and URIs to open specific plugins' settings pages, and allows you to jump to setting the hotkey for a plugin from the Command palette (using ctrl or cmd enter).

If any of these features have been added since 1.0, it's news to me -- and not listed in any Obsidian changelogs.

**Comment by Randy-Astle** [comment]
Good to know! :-)

I'll install it again in my vaults - and look at the other features. :-D 

Thanks,
